modules/ROOT/partials/configuration/autocorrect_autocorrect.adoc

```diff
@@ -0,0 +1,28 @@
+[[autocorrect_autocorrect]]
+== `+autocorrect_autocorrect+`
+
+This option enables or disables spelling error autocorrection when the {productname} editor is loaded.
+
+When set to `true`, spelling error autocorrection is enabled when {productname} is initialized.
+
+=== Usage note
```

Comment:
This change breaks the flow of information... the type, default and possible values no longer makes sense.

- [ ] Revert this change or come up with a different solution to prevent the mess with the flow of information

Reply:
I moved usage note to the end of both files.

This breaks the pattern of the Example being the last thing in these configuration documents. Which was my reason for the placement.

That said, putting the *Usage note* section at the end was my initial impulse. It’s an additional piece of information that makes sense only if the previous info is grokked.